crates/store/re_log_encoding/src/lib.rs

```diff
@@ -62,6 +78,10 @@ impl EncodingOptions {
         compression: Compression::LZ4,
         serializer: Serializer::MsgPack,
     };
+    pub const PROTOBUF: Self = Self {
```

Comment:
nit: might I suggest we rename these to ``MSGPACK_UNCOMPRESSED``, ``MSGPACK_COMPRESSED``, ``PROTOBUF_COMREPSSED``?